backend/src/main/java/com/google/backend/servlets/AlertVisualizationServlet.java

```diff
@@ -46,4 +54,39 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       throw new ServletException(e);
     }
   }
+  
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) 
+      throws ServletException, IOException {
+    String[] data = processRequestBody(request);
+    Long id = Long.parseLong(data[0]);
+    String priority = data[1];
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    try {
+      Entity alertEntity = datastore.get(KeyFactory.createKey(Alert.ALERT_ENTITY_KIND, id));
+      alertEntity.setProperty(Alert.PRIORITY_PROPERTY, priority);
+      datastore.put(alertEntity);
+    } catch (EntityNotFoundException e) {
+      throw new ServletException(e);
+    }
+  }
+
+  /** 
+    * Read from the request body, which contains data in the format "alertId statusToChangeTo".
+    * For example, the body could be "4785074604081152 RESOLVED".
+    */
+  private String[] processRequestBody(HttpServletRequest request) 
```

Comment:
Is there a reason why the data format is as what you mentioned (`alertId statusToChangeTo`) separated by the space?

Also as usual prefer structured as compared to unstructured data :D

Reply:
Oops I forgot to change this comment. I will change the comment
So yes you can pass an object to the backend, but I found it more difficult to deal with processing the object rather than just using this unstructured method.
In hindsight, I could have done this by passing in the data as parameters, because there are only two...